Kitodo/src/main/java/org/kitodo/services/data/FilterService.java

```diff
@@ -546,22 +550,24 @@ private QueryBuilder createBatchIdFilter(String filter, ObjectType objectType) t
      * @param taskStatus
      *            {@link TaskStatus} of searched step
      * @param negate
-     *            true or false, if true create simple queries with contains false
+     *            true or false, if true create simple queries with contains
+     *            false
      * @param objectType
      *            as {@link ObjectType}
      * @return query as {@link QueryBuilder}
      */
     private QueryBuilder createTaskFilters(String filter, FilterString filterString, TaskStatus taskStatus,
             boolean negate, ObjectType objectType) {
-        // extracting the substring into parameter (filter parameters e.g. 5, -5,
+        // extracting the substring into parameter (filter parameters e.g. 5,
+        // -5,
```

Comment:
Please adjust this line with the following line as your code formatter doe it wrong.

Reply:
Multiple line comments should be inside `/** ... */`